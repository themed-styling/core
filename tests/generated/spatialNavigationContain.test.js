import 'jest-styled-components'

import { spatialNavigationContain } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(spatialNavigationContain, 'spatialNavigationContain')
