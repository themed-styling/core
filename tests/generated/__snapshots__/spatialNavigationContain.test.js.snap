// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spatialNavigationContain renders 0 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: 0;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: 0;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: 0;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders function 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: has;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: has;
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders invalid theme path 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders negative number 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: July;
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: July;
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    spatial-navigation-contain: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    spatial-navigation-contain: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    spatial-navigation-contain: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    spatial-navigation-contain: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    spatial-navigation-contain: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: July;
}

@media May {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: September;
  }
}

@media January {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media June {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: July;
}

@media May {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: September;
  }
}

@media January {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media June {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: January;
  }
}

@media January {
  .c0 {
    spatial-navigation-contain: December;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: April;
  }
}

@media June {
  .c0 {
    spatial-navigation-contain: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: January;
  }
}

@media January {
  .c0 {
    spatial-navigation-contain: December;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: April;
  }
}

@media June {
  .c0 {
    spatial-navigation-contain: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    spatial-navigation-contain: July;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media January {
  .c0 {
    spatial-navigation-contain: September;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media June {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    spatial-navigation-contain: July;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media January {
  .c0 {
    spatial-navigation-contain: September;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: November;
  }
}

@media June {
  .c0 {
    spatial-navigation-contain: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    spatial-navigation-contain: January;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: December;
  }
}

@media January {
  .c0 {
    spatial-navigation-contain: April;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: May;
  }
}

@media June {
  .c0 {
    spatial-navigation-contain: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    spatial-navigation-contain: January;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: December;
  }
}

@media January {
  .c0 {
    spatial-navigation-contain: April;
  }
}

@media April {
  .c0 {
    spatial-navigation-contain: May;
  }
}

@media June {
  .c0 {
    spatial-navigation-contain: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders postitive number 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders string 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: July;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: July;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: June;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: June;
}

<div
    className="c0"
  />,
]
`;

exports[`spatialNavigationContain renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`spatialNavigationContain renders valid theme path 1`] = `
Array [
  .c0 {
  spatial-navigation-contain: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  spatial-navigation-contain: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
